pkg/types/ovirt/client.go

```diff
@@ -6,13 +6,13 @@ import (
 
 // GetConnection is a convenience method to get a connection to ovirt api
 // form a Config Object.
-func GetConnection(config Config) (*ovirtsdk.Connection, error) {
+func GetConnection(ovirtConfig Config) (*ovirtsdk.Connection, error) {
```

Comment:
`pkg/types` is API package for installer, so the consumers of it shouldn't force to vendor in SDKs...

the `pkg/types` should probably not have ovirt client dependencies..
is there is reason why these are moved from asset/installconfig to pkg/types

Reply:
> is there is reason why these are moved from asset/installconfig to pkg/types
It was in response to: https://github.com/openshift/installer/pull/3047#discussion_r374751508

I think a proper solution would be to just move the type and keep the rest of the logic in asset.

Reply:
I'm moving it back the original `pkg/asset/installconfig`